### ISSUE TITLE
Inaccessible share flow

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/phoenix#2593]
[This issue is a draft, collecting the accessibility issues of all components related to share/collaborators flow. This issue text will probably be edited several times, and where applicable, specialized, component-related issues will be opened.]

# New Collaborators autocomplete
* has no accessible name/label
* auto complete is not perceivable by screen readers

<img width="601" alt="Screenshot 2019-11-29 at 10 07 11" src="https://user-images.githubusercontent.com/505181/69857317-09f71600-1290-11ea-93c9-357fbe6cd472.png">

# Remove button in "selected collaborators" list is not keyboard accessible
* Is a span with a click event, needs to be a button

<img width="269" alt="Screenshot 2019-11-29 at 10 06 43" src="https://user-images.githubusercontent.com/505181/69857295-f77cdc80-128f-11ea-855a-d481562d3c4a.png">

# Edit and remove buttons in "collaborator overview panel" list is not keyboard accessible
* Is a span with a click event, needs to be a button

<img width="773" alt="Screenshot 2019-11-29 at 10 05 52" src="https://user-images.githubusercontent.com/505181/69857270-dae0a480-128f-11ea-9a15-921fef456d53.png">

# *Skipping roles select custom select for now*

# Invisible states of collaborators tab-panel can be reached via keyboard
* Check solution of adding inert and aria-hidden attributes on the container of the invisible state

**Comment by marcus-herrmann** [comment]
Corresponding PR: #2622
